docs/rst/manual/data_types/index.rst

```diff
@@ -277,6 +277,16 @@ takes 5 arguments:
 NBINS set the number of discrete values, and the other arguments have the same
 effect as in ERRF.
 
+TRIANGULAR
+----------
```

Comment:
Quick question; shouldn't it have the same markup for headline as the other? 
`,,,,,`

Reply:
Well spotted :+1: